packages/core/src/components/TrendLine/TrendLine.tsx

```diff
@@ -25,7 +26,7 @@ function color(data: number[]): string | undefined {
   return COLORS.STATUS.ERROR;
 }
```

Comment:
I specifically pulled these thresholds from Lighthouse itself. I think making color a `string |(data: number[]) => string` and having plugin-lighthouse provide this function maybe makes more sense?

Reply:
Not sure we even need a function as opposed to just passing the color prop directly, we're not calculating it multiple times anyway.  I think it would make sense to forward the `color` prop to the line component, and add a `lineStyle` prop that forwards to the line `style` prop.